src/lib/finishPollMessage.ts

```diff
@@ -1,21 +1,38 @@
-import { IModify, IPersistence, IRead } from '@rocket.chat/apps-engine/definition/accessors';
-import { RocketChatAssociationModel, RocketChatAssociationRecord } from '@rocket.chat/apps-engine/definition/metadata';
+import {
```

Comment:
There's a lot of tslint issues in this PR as well, please refer to my suggestion in previous PR for fixing it